commit ed04f56a66c2a8a7aadfda12b9534850bd6e57c1
Author: Volker Krause <vkrause@kde.org>
Date:   Mon Jan 1 14:51:00 2018 +0100

    Unexport kjscmd
    
    Summary:
    This is a user tool, not a build-time tool. Having it exported makes it
    required to be available (and runnable) for the corresponding find_package
    call to work, which is by default not the case when cross-compiling.
    
    Reviewers: #frameworks
    
    Tags: #frameworks
    
    Differential Revision: https://phabricator.kde.org/D9589

diff --git a/src/kjscmd/CMakeLists.txt b/src/kjscmd/CMakeLists.txt
index 95f7c31..5718d29 100644
--- a/src/kjscmd/CMakeLists.txt
+++ b/src/kjscmd/CMakeLists.txt
@@ -10,4 +10,4 @@ target_link_libraries(kjscmd5
     KF5::JsEmbed
 )
 
-install(TARGETS kjscmd5 EXPORT KF5JsEmbedTargets ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS kjscmd5 ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
